test/test-cases/multiple-routes/app/src/App.js

```diff
@@ -12,7 +23,7 @@ const Details = loadable(() => import('./handlers/Details'), {
 
 export default ({ site }) => (
   <Fragment>
-    <Route path="/" exact render={() => <Home site={site} />} />
-    <Route path="/details/:id" render={() => <Details site={site} />} />
+    <Route path={routes[site].home} exact render={() => <Home site={site} />} />
+    <Route path={routes[site].details} render={() => <Details site={site} />} />
```

Comment:
Feel like this kind of simple example should be added to the docs site for `sites`.